src/routes/index.svelte

```diff
@@ -20,12 +20,17 @@
 
   import Journey from '../lib/journey/journey.svelte';
   import { initialize, type InitObject } from '$journey/journey.store';
+  import { initializeContent } from '$lib/locale.store';
+
+  export let content;
```

Comment:
This `content` is provided by the "shadow endpoint" that matches this page component, for this, that would be `./index.ts`.